packages/editor/src/client/atoms/tag/selectTags.tsx

```diff
@@ -65,7 +65,10 @@ export function SelectTags({name, tagType, selectedTags, setSelectedTags}: Selec
       name={name}
       value={selectedTags}
       data={availableTags}
-      onChange={(value: string[]) => setSelectedTags(value)}
+      onChange={(value: string[]) => {
```

Comment:
Fixes a bug that you can type in whatever and press enter and you create a new item. That's a bug in the comments currently too.

creatable={false} does not work for w/e reason.